support-frontend/assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.jsx

```diff
@@ -286,16 +286,16 @@ function setUpPaymentListenerSca(
   paymentRequest.on('paymentmethod', ({ complete, paymentMethod, ...data }) => {
 
     const processPayment = () => {
-      if (paymentMethod && paymentMethod.card && paymentMethod.card.wallet) {
-        trackComponentEvents({
-          component: {
-            componentType: 'ACQUISITIONS_OTHER',
-          },
-          action: 'CLICK',
-          id: 'stripe-prb-wallet',
-          value: paymentMethod.card.wallet.type
-        })
-      }
+      const walletType = paymentMethod && paymentMethod.card && paymentMethod.card.wallet ? paymentMethod.card.wallet.type : 'no-wallet';
```

Comment:
can we use optional chaining + nullish coalescing e.g?

```js
const walletType = paymentMethod?.card?.wallet?.type ?? 'no-wallet';
```

Reply:
our version of flow doesn't appear to like it, I think I'll wait till we've migrated to TS!

Reply:
ahhh ofc! I was too hopeful there 😂